spring-cloud-loadbalancer/src/main/java/org/springframework/cloud/loadbalancer/annotation/LoadBalancerClient.java

```diff
@@ -67,4 +67,6 @@
 	 */
 	Class<?>[] configuration() default {};
 
+	Class<?>[] lifecycleProcessors() default {};
```

Comment:
Just get lifecycle processors from NamedContextFactory (ie from the `configuration` attribute) so this new field can be removed.